tests/test_examples.py

```diff
@@ -19,23 +19,27 @@
 
 logger = logging.getLogger(__name__)
 
+CONFIG = dict(
+    from_ts=dateutil.parser.isoparse("2014-07-01T00:10:00+00:00"),
+    to_ts=dateutil.parser.isoparse("2015-01-01T00:00:00+00:00"),
+    tag_list=["asgb.19ZT3950%2FY%2FPRIM", "asgb.19PST3925%2FDispMeasOut%2FPRIM"],
```

Comment:
need fake tags in this great open-source project of ours.

Reply:
This was a refactor, but you mean make this "faktag1", etc., right?

Reply:
Ok, I just assumed `asgb.19ZT...` refers to an actual tag. Make it whatever you want.